sorc/ncep_post.fd/MISCLN.f

```diff
@@ -169,11 +173,34 @@ SUBROUTINE MISCLN
       integer ISTART,ISTOP,JSTART,JSTOP,MIDCAL
       real    dummy(IM,jsta:jend)
       integer idummy(IM,jsta:jend)
+!     NEW VARIABLES USED FOR EFFECTIVE LAYER
+      INTEGER,dimension(:,:),allocatable :: EL_BASE, EL_TOPS
+      LOGICAL,dimension(:,:),allocatable :: FOUND_BASE, FOUND_TOPS
+      INTEGER,dimension(:,:),allocatable :: L_THETAE_MAX
+      INTEGER,dimension(:,:),allocatable :: CAPE9, CINS9
+      CHARACTER(LEN=5)   :: IM_CH, JSTA_CH, JEND_CH, ME_CH
+      CHARACTER(LEN=60)  :: EFFL_FNAME
+      CHARACTER(LEN=60)  :: EFFL_FNAME2
+      INTEGER            :: IREC, IUNIT
+      INTEGER            :: IREC2, IUNIT2
+      LOGICAL            :: debugprint
+      INTEGER            :: EL_SCHEME
+      INTEGER            :: LLL
+      INTEGER            :: LLCL_PAR, LEQL_PAR
+      REAL               :: LMASK, PSFC, CAPE_PAR, CINS_PAR, LPAR0
+      REAL, DIMENSION(4) :: PARCEL0
+      REAL, DIMENSION(:), ALLOCATABLE  :: TPAR_B, TPAR_T   
+      REAL, DIMENSION(:), ALLOCATABLE  :: TPAR_TMP 
+      REAL, DIMENSION(:), ALLOCATABLE  :: P_AMB, T_AMB, Q_AMB, ZINT_AMB
+      REAL, DIMENSION(:,:,:), ALLOCATABLE  :: TPAR_BASE, TPAR_TOPS
 
 !     
 !****************************************************************************
 !     START MISCLN HERE.
 !     
+       debugprint = .TRUE.
```

Comment:
debugprint was used during the code development to print out more information for special tracing & debugging the scheme 1 and 2 for searching the effective-layer. It is not necessary for routine use. 
I suggest to set "debugprint = .FALSE." as default.